pgx-pg-sys/build.rs

```diff
@@ -164,8 +164,19 @@ fn main() -> eyre::Result<()> {
                     .join(", ")
             )
         })?;
-        let specific = pgx.get(&found_feat)?;
-        vec![(found_ver, specific)]
+
+        if PgConfig::is_in_environment() {
```

Comment:
Might be cleaner to just do `if let Some(...) = PgConfig::from_env()` rather than changing it to return an error only if that's not checked.

Reply:
`if let Ok(...) = PgConfig::from_env()` you mean?  I can do that.  I changed it to return a Result and ought to treat it as such, I suppose.

Reply:
Done in [cac55e4](https://github.com/tcdi/pgx/pull/1020/commits/cac55e477996b0492b0941be14e24f4088f14bae)